src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -39,7 +40,7 @@ public class PersonCard extends UiPart<Region> {
     @FXML
     private Label remark;
     @FXML
-    private FlowPane assignedTier;
+    private FlowPane assignedStatusAndTier;
```

Comment:
```suggestion
    private FlowPane assignedTier
    @FXML
    private FlowPane assignedStatus
```
Perhaps we can isolate the logic if possible and make the appropriate changes in the CSS file, which would improve maintainability. Feel free to keep it as is if there are issues related to difficult UI logic.

Reply:
Hmm i'll try to refactor it, it's just that it may appear vertically instead of horizontally side by side.